requests/__init__.py

```diff
@@ -70,6 +71,12 @@ def check_compatibility(urllib3_version, chardet_version):
     assert minor < 1
     assert patch >= 2
 
+# Check for older version of pyOpenSSL
+def __check_pyopenssl(pyopenssl_version, cryptography_version):
+    # Warning only applies for Cryptography < 1.3.4
+    if list(map(int, cryptography_version.split('.'))) < [1, 3, 4]:
+        # Warning only applies for pyOpenSSL < 16.0.0
```

Comment:
So we don't really care about the version of pyOpenSSL, the only reason it's significant is 16.0 is where a version constraint for cryptography > 1.0 is put in place. Let's remove this check and just launch the warning if cryptography doesn't meet our requirement. That's the source of the slowdown.

Reply:
Ah, I just misunderstood the cause of the slowdown. Thanks for pointing that out, I'll get on this as soon as I can!